pkgs/by-name/cc/ccid/package.nix

```diff
@@ -23,13 +25,13 @@ stdenv.mkDerivation rec {
   postPatch = ''
     patchShebangs .
     substituteInPlace src/Makefile.am --replace-fail /bin/echo echo
+    substituteInPlace meson.build --replace-fail \
+      "pcsc_dep.get_variable('usbdropdir')" \
+      "'$out/pcsc/drivers'"
   '';
 
-  configureFlags = [
-    "--enable-twinserial"
-    "--enable-serialconfdir=${placeholder "out"}/etc/reader.conf.d"
-    "--enable-ccidtwindir=${placeholder "out"}/pcsc/drivers/serial"
-    "--enable-usbdropdir=${placeholder "out"}/pcsc/drivers"
```

Comment:
Do we know that these don’t matter any more? I see that the `usbdropdir` is handled, but what about the others?

Reply:
The first option is not needed as it is enabled by the meson flag below.
The other two are handled by the substitute.